valid.go

```diff
@@ -68,8 +68,13 @@ func (e *TagsMdError) Error() string {
 	return "cbor: CBOR tag isn't allowed"
 }
 
-// valid checks whether CBOR data is complete and well-formed.
-func (d *decoder) valid() error {
+// Valid checks whether the CBOR data is complete and well-formed.
+func (d *decoder) Valid() error {
```

Comment:
Because `decoder` is an unexported struct, can we make its function unexported?

Reply:
good point, done in 5d9d2d5